### ISSUE TITLE
Application crashes when openend for a while

### ISSUE THREAD

**Issue by Cypher87** [julianpaulozzi/LiteDbExplorer#36]
**This is a:**
Bug Report

**Version:**
0.8.4.3

**Describe the bug**
I have a LiteDB database file on a network share. I have a collection opened. After a while (not sure what the timeframe is, this happend after an hour or so) when I hit the refresh button the application will hang.

**Steps to reproduce:**
1. Open a LiteDB database file from a share
2. Open a collection
3. Keep the app running in the background for x amount of time
4. Hit the refresh button

**What is expected:**
Data will be refreshed

**What is actually happening:**
The app hangs and crashes
"The program LiteDbExplorer.exe version 0.8.4.3 stopped interacting with Windows and was closed"

**Comment by julianpaulozzi** [comment]
Do you share this file with other processes?

**Comment by julianpaulozzi** [comment]
I did a little research about this case (file over network).
My recommendation, do not do this!
If you take a quick look you will see that even Windows Explorer hangs many on this case.

In this GUI once the database is opened it is always connected (never disposes until closed) and all references are always in memory and directly 2 way binded with UI. There is no reconnection or resilience mechanism.

Anyway I added an alert when detected file over network and changed the connection way to ensure exclusive access.
